devices.json

```diff
@@ -0,0 +1,14 @@
+[
+  {
+    "id": "Nexusplayer",
+    "adb": {
+      "ro.product.device": [ "fugu" ],
+      "ro.bootloader": ["FUGU-02.11"],
+      "ro.build.id": ["MMB29T", "LMY48M", "LMY47D"]
```

Comment:
I get them from nexus player which is 6.0.1 version via adb
[ro.product.device]: [fugu]
[ro.bootloader]: [FUGU-02.11]
[ro.build.id]: [MMB29T]